smallcloudai/refact-chat-js
# Fix: Copy button in the web

# Fix: Copy button in the web
## Description
- Problem: Copy button was appearing on the left side, moving right the code sample.
- Solution: Adjusted layout for Pre element for web.
- [Any background context or related links?](https://github.com/orgs/smallcloudai/projects/5/views/1?filterQuery=alashchev17&pane=issue&itemId=76108737)

## Type of change

## How to Test

- Step 1: Start new chat
- Step 2: Ask a model to write any code
- Step 3: See the result

## Screenshots
### Code sample with adjusted button
![Code sample with adjusted button](https://i.imgur.com/SkLVxWr.png)

## Checklist
